plugin.video.tweakers/resources/language/English/strings.po

```diff
@@ -13,33 +13,11 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Language: en\n"
-"Plural-Forms: nplurals=2; plural=(n != 1)\n"
-
-msgctxt "Addon Summary"
-msgid "Watch tech videos from Tweakers.net (dutch)"
-msgstr ""
-
-msgctxt "Addon Description"
-msgid "Watch tech videos from Tweakers.net (dutch)"
-msgstr ""
-
-msgctxt "Addon Disclaimer"
-msgid "For bugs, requests or general questions visit the tweakers.net thread on the XBMC forum."
-msgstr ""
-
-#
-     * Main
```

Comment:
In case I was not clear enough: I only meant those lines starting with "*", not the other entries here. ;)
If you want to use comments,  the line has to start with a "'#".